.env.example

```
@@ -0,0 +1,75 @@
+ANNUAL_COST_EMPLOYEE=650000
```

I used your `.env` and the `APP` variable was missing.
```suggestion
ANNUAL_COST_EMPLOYEE=650000
APP=my-app
```